CONTRIBUTING.md

```diff
@@ -157,6 +179,12 @@ dbg.tcpListen("localhost", 9966)
 5. Start Path of Building Community
 6. Attach the debugger
 
+#### Miscellaneous tips
+If you're on windows, consider downloading [git for windows](https://git-scm.com/downloads) and installing git bash. Git bash comes with a variety of typical linux tools such as grep that can make navigating the code base much easier.
+
+If you're using linux you can run the `./runtime/Path{space}of{space}Building.exe` executable with wine. You will need to provide a valid wine path to the emmy lua debuger directory.
```

Comment:
If you're using linux you can run the `./runtime/Path{space}of{space}Building.exe` executable with wine. You will need to provide a valid wine path to the emmy lua debugger directory.

(spelling)